client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+{
+    "common": {
+        "ok": "OK",
+        "submit": "Übernehmen",
+        "cancel": "Abbrechen",
+        "overwrite": "Überschreiben",
+        "continue": "Weiter",
+        "dismiss": "Abbrechen",
+        "prompt": "Hinweis",
+        "warning": "Warnung",
+        "attention": "Achtung!",
+        "confirm": "Bestätigen",
+        "close": "Schließen",
+        "save": "Speichern",
+        "cancelled": "Abgebrochen",
+        "enabled": "Eingeschaltet",
+        "commandOK": "Befehl erfolgreich übertragen!",
+        "commandFailed": "Es ist ein Fehler aufgetreten:",
+        "connectionError": "Verbindungsfehler",
+        "m2": "m²",
+        "hours": "Stunde",
+        "hours_plural": "Stunden",
+        "months": ["Jan","Feb","Mär","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dez"],
+        "days": ["Montag","Dienstag","Mittwoch","Donnerstag","Freitag","Samstag","Sonntag"]
+    },
+    "navi": {
+        "home": "Home",
```

Comment:
since this ought to be the link to the valetudo start page:
"home": "Startseite",